invenio_administration/assets/semantic-ui/js/invenio_administration/src/components/utils.js

```diff
@@ -9,8 +9,8 @@ export const sortFields = (schema) => {
     Object.entries(schema)
       // sort by order
       .sort((a, b) => {
-        if (a[1]?.ui && b[1].ui) {
-          return a[1].ui.order > b[1].ui.order;
+        if (a[1] && b[1]) {
+          return a[1].order - b[1].order;
```

Comment:
Is this change needed?

Reply:
Yes, it's a sorting using a comparison function. See:https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#:~:text=copy%20is%20made.-,Description,-If%20compareFn%20is

Reply:
Thanks for pointing this out!
It looks like the `a - b` and the `a > b` comparison functions do the same thing, however, the `sort()` function excepts a comparison function, which has to return a number, otherwise it performs a cast to a number, which doesn't look like we expect. See the link: https://stackoverflow.com/questions/24080785/sorting-in-javascript-shouldnt-returning-a-boolean-be-enough-for-a-comparison (both answers are useful).